docs/retrieve.txt

```diff
@@ -0,0 +1,360 @@
+to ``where()``:
+
+- ``skip()``: sets the number of documents to skip when returning results
+- ``take()``: sets the total number of documents to return
+- ``first()``: returns the first document that matches the query filter
+
+Skip and Limit Results
+~~~~~~~~~~~~~~~~~~~~~~
+
+The following example queries for documents in which the ``year`` value is ``1999``.
+The operation skips the first ``2`` matching documents and outputs a total of ``3``
+documents.
+
+.. important:: Required File Changes
+
+   To run the code example, you must edit the ``MovieController.php`` file in your
+   Laravel Quick Start application. Copy the code example and paste it into the ``show()``
+   function, replacing any existing code inside this function.
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: php
+
+      return view('browse_movies', [
+         'movies' => Movie::where('year', 1999)
+            ->skip(2)
+            ->take(3)
+            ->get()
+      ]);
+
+   .. output::
+      :language: none
+      :visible: false
+
+      Title: Three Kings
+      Year: 1999
+      Runtime: 114
+      IMDB Rating: 7.2
+      IMDB Votes: 130677
+      Plot: In the aftermath of the Persian Gulf War, 4 soldiers set out to steal gold
+      that was stolen from Kuwait, but they discover people who desperately need their help.
+
+      Title: Toy Story 2
+      Year: 1999
+      Runtime: 92
+      IMDB Rating: 7.9
+      IMDB Votes: 346655
+      Plot: When Woody is stolen by a toy collector, Buzz and his friends vow to rescue him,
+      but Woody finds the idea of immortality in a museum tempting.
+
+      Title: Beowulf
+      Year: 1999
+      Runtime: 95
+      IMDB Rating: 4
+      IMDB Votes: 9296
+      Plot: A sci-fi update of the famous 6th Century poem. In a beseiged land, Beowulf must
+      battle against the hideous creature Grendel and his vengeance seeking mother.
+
+.. _laravel-retrieve-one:
+
+Retrieve Only One Result
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To retrieve only one document that matches a set of criteria, use the ``where()`` method
+followed by the ``first()`` method. By default, the ``first()`` method returns the first
+matching document according to the documents' natural order, or as they appear in the
+database. If you apply a sort to the find operation, ``first()`` returns the first matching
+document according to the sorted order. 
+   
+The following example returns one document in which the value of the ``runtime`` field
+is ``30``.
+
+.. important:: Required File Changes
+
+   To run the code example, you must edit the ``MovieController.php`` file in your
+   Laravel Quick Start application. Copy the code example and paste it into the ``show()``
+   function, replacing any existing code inside this function.
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: php
+      
+      return view('browse_movies', [
+         'movies' => Movie::where('runtime', 30)
```

Comment:
You should set the sorting order. Otherwise the order is unpredictable. Except if we know there is only one result (unique constraint).

```php
$movies = Movie::where('runtime', 30)->orderBy('_id');
```